cmd/earthly/account_cmds.go

```diff
@@ -295,6 +303,25 @@ func (app *earthlyApp) actionAccountRegister(cliCtx *cli.Context) error {
 	return nil
 }
 
+func promptPassword() (string, error) {
+	fmt.Printf("pick a password: ")
```

Comment:
Should we capitalize these? Asking because some other prompts are capitalized.

Reply:
good question!

sudo, for instance, uses lower case:

```
sudo whoami
[sudo] password for alex: 
```

where as 'adduser` (under ubuntu), uses upper case:

```
docker run --rm -ti ubuntu /bin/sh -c 'adduser testuser'
Adding user `testuser' ...
Adding new group `testuser' (1000) ...
Adding new user `testuser' (1000) with group `testuser' ...
Creating home directory `/home/testuser' ...
Copying files from `/etc/skel' ...
New password: 
```

so much for looking at standard cli practices for inspiration :laughing: 

I'll capitalise them.